etcdctl/ctlv3/command/printer.go

```diff
@@ -43,6 +43,7 @@ type printer interface {
 	MemberList(v3.MemberListResponse)
 
 	EndpointStatus([]epStatus)
+	MoveLeader(leader, target uint64)
```

Comment:
since this is an RPC, it should probably be `MoveLeader(leader, target uint64, v3.MoveLeaderResponse)` and support printerRPC